api/src/main/java/org/ehcache/spi/serialization/Serializer.java

```diff
@@ -26,7 +26,17 @@
 
 /**
  * Interface defining the contract used to transform types in a serial form.
- * Implementations must be thread-safe and must define a constructor accepting a single ClassLoader parameter.
+ * Implementations must be thread-safe and must define either or both of the two constructors:
```

Comment:
Admittedly I made the same mistake here, but the constructor details here are part of the DefaultSerializationProvider contract and not an inate part of the Serializer contract.